api/main.py

```diff
@@ -101,6 +101,33 @@ async def get_node(node_id: str):
     return await db.find_by_id(Node, node_id)
 
 
+@app.get('/nodes')
+async def nodes():
+    return {'nodes': await db.find_all(Node)}
+
+
+@app.get('/node/{attr_name}/{attr_value}', response_model=Node)
+async def get_node(attr_name: str, attr_value: str):
+    return await db.find_by_attribute(Node, attr_name, attr_value)
+
+
+@app.get('/node/revision/{attr_name}/{attr_value}', response_model=Node)
+async def get_node(attr_name: str, attr_value: str):
+    return await db.find_by_attribute(Node, "revision", attr_name, attr_value)
+
+
+@app.get('/node', response_model=Node)
```

Comment:
The response should be a list of `Node` objects, so I believe that should be reflected into the `response_model` value.  See the docs: https://fastapi.tiangolo.com/tutorial/response-model/

> It receives the same type you would declare for a Pydantic model attribute, so, it can be a Pydantic model, but it can also be, e.g. a list of Pydantic models, like List[Item].